lib/surface_site_web/live/data.ex

```diff
@@ -51,6 +51,13 @@ defmodule SurfaceSiteWeb.Data do
                 decrement/increment the counter's value.
               </#Markdown>
 
+              <#Markdown>
+                > **Note**: The `data` macro is also available in `Surface.Component`
+                even if that kind of component does not have state.
+                It allows you to define intermediary assigns in the `update` function of the component
+                that computes values based on one or several properties.
```

Comment:
By the way, is `update` highlighted anywhere in the docs? I don't think I've seen it.